Week3/homework/step2-3.js

```diff
@@ -4,10 +4,12 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  if (num > 0) {
+    for (num > 0; num--; ) {
+      result += str;
+    }
+    console.log(str, num, result);
+  }
```

Comment:
Couple things here:
1. The `for` loop conditions again seem a bit complicated than it should be. You could again simply write it as: `for(let i = num; i > 0; i--) { // loop body }`
2. The `console.log` need not be there.

Reply:
1. I defined "num" instead of "i" but not problem I can change.
2. console.log just for checking on the browser. I will delete it